tools/serverpod_cli/lib/src/test_util/compilation_unit_helpers.dart

```diff
@@ -129,10 +130,17 @@ abstract class CompilationUnitHelpers {
     CompilationUnit unit, {
     required String uri,
   }) {
-    return unit.directives
-        .whereType<PartDirective>()
-        .where((directive) => directive.uri.stringValue == uri)
-        .firstOrNull;
+    return unit.directives.whereType<PartDirective>().where((directive) {
+      String directiveUri = directive.uri.stringValue!;
+
+      // Windows-specific: separator fix
+      if (Platform.isWindows) {
```

Comment:
Suggestion: Break this out into a shared helper between the methods.

Also add documentation and possibly a link go the bug that you created around why this is needed.